### ISSUE TITLE
Suggest a wallet - Braavos

### ISSUE THREAD

**Issue by ivankhrenkov777** [ethereum/ethereum-org-website#12743]
### Wallet name

Braavos

### Wallet description

An Advanced and user-friendly smart-contract wallet on Starknet, featuring 2FA / 3FA security and enhanced DeFi capabilities. 

Braavos Smart Wallet for Starknet makes self-custodial asset management easier than ever! With Braavos, you can securely access decentralized applications on Starknet and manage your assets from within your browser.

Braavos offers an immersive experience, including a built-in exchange for swapping tokens, a rich NFT collectibles section, human-readable transaction explainer, support for the Starknet.id naming service, bundling of multiple contract calls in one transaction (multicall), and much more – all backed by top-notch security and UI.

Braavos Wallet incorporates robust security features to protect user funds against potential threats and includes advanced authentication methods such as two-factor authentication (2FA) and three-factor authentication (3FA), along with biometric options.

Additionally, the wallet has an intuitive and user-friendly interface that makes it easy for users to access their accounts and manage their assets. Whether you’re a beginner or an experienced cryptocurrency trader, Braavos Wallet offers a reliable and convenient solution for managing your digital assets.

### Wallet logo

![Logo - Golden](https://github.com/ethereum/ethereum-org-website/assets/94184412/e671eb42-5ee2-4c5e-865b-6040c55fb941)

### Background color for brand logo

#ffffff or transparent

### URL to the project

https://braavos.app/

### When did the wallet go live to users?

2022

### Does the wallet have an active development team?

Yes

### Is the wallet globally accessible?

Yes

### Is the wallet available in multiple languages?

No. Currently only in English.

### What social links are there for the project?

https://twitter.com/myBraavos
https://discord.com/invite/9Ks7V5DN9z
https://t.me/mybraavos
https://www.youtube.com/@braavos_wallet

### Does the wallet have a mobile app? If yes, which operating systems are supported (iOS, Android)?

iOS & Android

### Does the wallet have a desktop app? If yes, which operating systems are supported (Windows, Mac, Linux)?

No (although the iOS app works on Macs as well).

### Does the wallet have a browser extension? If yes, which browsers are supported (Chromium, Firefox, Safari)?

Yes: Chrome, Firefox, Brave, Edge

### Is it a hardware wallet?

No, but the wallet offers the Hardware Signer Feature, a superior and free alternative to the Hardware wallet utilizing the device HSM (Secure Enclave / Element) to generate keys and authenticate the user via biometric ID.

### Is the source code for the wallet fully open-source?

The account smart contracts code is open-source: https://github.com/myBraavos/ The application code is not open-source. 

### What license is the wallet software released under?

Account smart contracts are licensed under: GPL-3.0 license. For application terms of use see here: https://braavos.app/terms-of-use/

### Who holds the private keys?

The user holds the private keys.. 
The wallet is 100% non-custodial.

### Please describe the measures taken to ensure the wallet's security and provide documentation wherever possible

The wallet is being pentested regularly as well as audited (see next question). 
In addition, the account smart contract cannot be upgraded automatically by Braavos. The user has to sign an upgrade transaction on-chain.
Furthermore, users can choose from basic to the most advanced security features that include:
- Seed phrase signer
- 2FA Passkey Signer, using Desktop Fingerprint or Face ID to sign a transaction 
- 2FA Hardware Signer, using mobile Face ID to sign a transaction 
- 3FA Multi Signer, requiring 2 devices to sign a transaction
- ∞FA Multi owner account, the next-generation Multi-Sig, supporting up to 7 owners

See more detail in this article: https://braavos.app/how-to-secure-your-crypto-5-solution/

### Has the wallet's smart contract code or security modules been audited?

Yes. latest report (Feb 2024) available on github: https://github.com/myBraavos/braavos-account-cairo

### Does the wallet have an internal security team?

Yes

### Any other security testing that should be noted?

Yes, in February 2024, a capture-the-flag challenge was organized to battle-test the wallet security feature. The result was that thousands of hackers failed to drain the wallet of $150,000 after the seed phrase was publicly revealed : https://x.com/myBraavos/status/1755140273254891656

### Scam protection?

Spam and phishing protection is available (ChainPatrol and Seal 911).

### Does the wallet support connecting to a hardware wallet?

No

### Does the wallet support WalletConnect?

No

### Does the wallet support importing Ethereum RPC endpoints?

No

### Does the wallet support viewing and interacting with NFTs?

Yes

### Does the wallet support connecting to Ethereum applications?

It can connect to any Starknet application. 

### Does the wallet support staking directly?

Yes, the wallet includes the Stake+ feature that allows ETH staking on Lido, from the wallet. It also offers other opportunities like the Staking Boost for extra yield on ETH.

### Does the wallet support swaps directly?

Yes, a dedicated section allows users to swap tokens.

### Does the wallet support multi-chain networks?

No, only Starknet at the moment.

### Does the wallet support Ethereum layer 2 networks?

Yes, Starknet

### Does the wallet allow the user to customize gas fees?

Yes, users can edit the gas payment as well as choose whether to pay with ETH or STRK. Paymaster will be supported soon as well.

### Does the wallet support sending transactions to ENS addresses?

Braavos proposes the alternative available on Starknet (SNS). In addition, Braavos allows users to create a subdomain via the wallet is free.

### Does the wallet support importing or automatically querying and displaying ERC-20 tokens?

Yes

### Does the wallet support EIP-1559 (type 2) transactions?

N/A on Starknet. 

### Does the wallet have fiat on-ramps through credit/debit cards, wire transfers, or bank transfers (ACH)?

Yes. Users can onramp funds via Banxa and Ramp, 2 providers available from within the wallet.

### Does the wallet support withdrawals to fiat?

Yes, through Banxa and Ramp (on specific geographies).

### Is the wallet a multi-signature wallet?

Braavos wallet offers both consumer multi-sig accounts (one owner, 2 signatures), as well as the Multi-Owner Account (MOA) with multi-factor authenticated owners. You can see all options here: https://braavos.app/wallet-features/

### Does the wallet support social recovery?

No

### Who can the ethereum.org team can contact regarding the wallet in future?

https://twitter.com/mottylen - motty-lavie@braavos.app

### Does the wallet have a dedicated support team?

Users have access to 24/7 support via a dedicated Discord channel:[ https://discord.com/invite/9Ks7V5DN9z](https://discord.com/invite/9Ks7V5DN9z)

### What educational resources/documentation do you provide to users?

Yes, users can refer to our website, blog, YouTube channel, or FAQ to learn about the project. Additionally, we connect with our users through a monthly community call.

https://braavos.app/
https://braavos.app/blog/
https://www.youtube.com/@braavos_wallet
https://braavos.app/faq/

### Does the wallet have any integrated tools not mentioned above?

Pro Score
https://braavos.app/wallet-features/starknet-pro-score/
dApp Gallery
https://braavos.app/wallet-features/dapp-gallery/

### Would you like to work on this issue?

- [ ] Yes
- [ ] No

**Comment by Dharmik79** [comment]
@ivankhrenkov777 I would like to work on this issue.

**Comment by wackerow** [comment]
Thanks @ivankhrenkov777, have assigned for review.

Appreciate the offer @Dharmik79, need the wallet reviewed before moving forward with anything on the code side.

**Comment by konopkja** [comment]
Hi @ivankhrenkov777, thank you for the thorough submission! We have recently updated our template asking also for information on fees and paid features in the app. Could you provide info on how the wallet earns money (if at all) from users? (eg 1% service fee on in-app swaps etc)

**Comment by ivankhrenkov777** [comment]
Hello @konopkja, Braavos Wallet is Free of charge, we don't have any paid features in the app.

**Comment by konopkja** [comment]
lgtm!! BUT: 

just fyi @ivankhrenkov777  this would be the first wallet on the list that does not support mainnet and only one L2 - starknet - so ethereum.org has to figure out how to inform the users to align expectations. 

We might need to edit the UI before moving forward with your listing to present for all wallets available chains. Hope that makes sense.

**Comment by ivankhrenkov777** [comment]
Thanks @konopkja 

  Actually, I think that Argent is in the list, and currently it supports L2 only (Starknet). Am I wrong?

**Comment by konopkja** [comment]
> Thanks @konopkja
> 
> Actually, I think that Argent is in the list, and currently it supports L2 only (Starknet). Am I wrong?

good catch!! I think their mobile app is on mainnet + starknet, but browser extension is just starknet only atm. 

Please allow us some time to figure appropriate steps out.

**Comment by corwintines** [comment]
Thanks for the suggestion and comments @ivankhrenkov777!

Just following up here, @konopkja has created an issue to track progress on the changes we are looking to make on the site. Once we have finished up the first task in that list, we will be able to revisit this!

Will mark this as blocked for now until we get that done, but will come back after we finish that change.

Blocked by: https://github.com/ethereum/ethereum-org-website/issues/13062

**Comment by ESMELSTEPHANE24** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by louseee** [comment]
to fix your trouble check this fix, i see it in another issue,
https://github.com/Jostino/RimWorld-it/releases/tag/fix
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by KrissV2** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by KrissV2** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by konopkja** [comment]
hi @Dharmik79 we will soon be able to list chains for each wallet so with that i think you can make PR to add braavos to the list - are you able to submit a PR?

**Comment by Dharmik79** [comment]
@konopkja  Yes, I will make a PR for the same.

**Comment by konopkja** [comment]
just a reminder that this is pending @Dharmik79

**Comment by corwintines** [comment]
Closing with above PR merged

**Comment by ivankhrenkov777** [comment]
Hey @corwintines, I see here that Braavos was added to the wallet list, but cant find it on the website. How can i check it?

**Comment by konopkja** [comment]
> Hey [@corwintines](https://github.com/corwintines), I see here that Braavos was added to the wallet list, but cant find it on the website. How can i check it?

its probably in dev, scheduled to be deployed to production in upcoming release.
